src/styles/index.ts

```diff
@@ -4164,6 +4173,125 @@ const styles = (theme: ThemeColors) =>
             borderColor: theme.icon,
         },
 
+        videoPlayerControlsContainer: {
+            position: 'absolute',
+            bottom: CONST.VIDEO_PLAYER.CONTROLS_POSITION.NORMAL,
+            left: CONST.VIDEO_PLAYER.CONTROLS_POSITION.NORMAL,
+            right: CONST.VIDEO_PLAYER.CONTROLS_POSITION.NORMAL,
+            backgroundColor: theme.videoPlayerBG,
+            borderRadius: 8,
+            flexDirection: 'column',
+            overflow: 'visible',
+        },
+
+        videoPlayerControlsButtonContainer: {
+            flexDirection: 'row',
+            alignItems: 'center',
+            justifyContent: 'space-between',
+        },
+
+        progressBarOutline: {
+            width: '100%',
+            height: 5,
+            borderRadius: 10,
```

Comment:
Designers like multiples of 4. Were those numbers provided by the designer?

Reply:
Good catch! According to the [Figma](https://www.figma.com/file/ucwSmgVnBPRbe6srOG5KxG/Video-Player?node-id=29%3A10160&mode=dev) progress bar should have height 4px and border radius 2px. cc @kowczarz